project_role/README.rst

```diff
@@ -0,0 +1,74 @@
+=============
```

Comment:
Can you explain how to configure and use the module in readme, please.

Reply:
@tarteo thanks for feedback, added _Usage_ section,